src/patternfly/components/Content/content.scss

```diff
@@ -208,10 +208,13 @@
 
   small {
     display: block;
-    margin-bottom: var(--pf-c-content--small--MarginBottom);
     font-size: var(--pf-c-content--small--FontSize);
     line-height: var(--pf-c-content--small--LineHeight);
     color: var(--pf-c-content--small--Color);
+
```

Comment:
Couldn't you just remove the `margin-bottom` from `small` here, and it will get a margin bottom from L122?

Reply:
Oh my bad didn't realize they're the same size

Reply:
Doh! I should have thought about that a little more. Technically it's a breaking change now, where the way you did it before was not, even though it's redundant. How about we do it the way you did it originally, the create a follow up issue with a breaking change label just to remove the margin from `small`?

Reply:
Yep that sounds good, created this issue #1844 and updated this PR